util/security/security.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
```

Comment:
Do you think it's a better idea to rename it to `sem`? So that we would have:

```
sem.Enable()
```
Compared with
```
security.Enable()
```
(I think `security.Enable()` may be confusing to the developers)

Reply:
I named it as sem to avoid the acronym. But I am find to rename it, you can usually tell what the acronym means from context, such as `if sem.Enabled() && followup privilege check..`.